README.md

```diff
@@ -123,6 +123,7 @@ For edit access, get in touch on
 * [Josh Suereth](https://github.com/jsuereth), Google
 * [Reiley Yang](https://github.com/reyang), Microsoft
 * [WenTao Ou](https://github.com/owent), Tencent
+* [Marc Alff](https://github.com/marcalff), Oracle
```

Comment:
Putting it after JoshS (alphabetical ordering).

Reply:
Thanks, now fixed.